### PR TITLE
toggle single/double

### PR DESCRIPTION
**Pull request by tajas20006**
added "toggle" button to toggle between single/double translation.
need to start gtrans with -d option.

### CODE REVIEW

File: python/window.py [hunk 1]
```diff
@@ -26,6 +26,8 @@ def __init__(self, qsettings, src_lang, tgt_lang, middle_lang=None, title='Gtran
         self._init_gui()
         self._init_candidate_list()
 
+        self.setStyleSheet("QTextEdit{font-family: 'Ricty Diminished'; font-size: 28px;}")
```
Comment (takiyu) [inline python/window.py]: Is not it environment specific?

Comment (tajas20006) [reply python/window.py]: My fault. It shouldn't be there.

Comment (takiyu) [reply python/window.py]: Please add a commit to remove the code.

**Comment by takiyu** [comment]
Thanks. Merged.

**Pull request merged**